suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -12,7 +12,7 @@ export const networks = {
             tx: 'https://btc1.trezor.io/tx/',
             account: 'https://btc1.trezor.io/xpub/',
         },
-        features: ['rbf', 'sign-verify', 'amount-unit'],
+        features: ['rbf', 'sign-verify', 'amount-unit', 'coin-control'],
```

Comment:
has to be added in taproot features too, otherwise it will not work for taproot accounts